### PR TITLE
p2-rctl has learned how to delete Rolling Updates

### PR DESCRIPTION
**Pull request by rudle**

### CODE REVIEW

File: bin/p2-rctl/main.go [hunk 1]
```diff
@@ -82,6 +83,9 @@ var (
 	rollNeed                = cmdRoll.Flag("minimum", "minimum number of healthy replicas during update").Required().Short('m').Int()
 	rollPagerdutyServiceKey = cmdRoll.Flag("pagerduty-service-key", "Pagerduty Service Key to use for alerting if provided").String()
 
+	cmdDeleteRoll = kingpin.Command(cmdDeleteRollText, "Delete a rolling update.")
+	deleteRollID  = cmdDeleteRoll.Flag("id", "replication controller uuid").Required().Short('i').String()
```
Comment (mpuncel) [inline bin/p2-rctl/main.go]: s/replication controller/rolling update

**Comment by mpuncel** [comment]
👍

File: bin/p2-rctl/main.go [hunk 2]
```diff
@@ -211,6 +217,13 @@ func (r rctlParams) Delete(id string, force bool) {
 	r.logger.WithField("id", id).Infoln("Deleted replication controller")
 }
 
+func (r rctlParams) DeleteRollingUpdate(id string) {
+	err := r.rls.Delete(roll_fields.ID(id))
```
Comment (petertseng) [inline bin/p2-rctl/main.go]: does this delete labels too? yes it does, fantastic.

Comment (mpuncel) [inline bin/p2-rctl/main.go]: lol

**Pull request merged**